migrate/20231019_rotate_pg_certificates.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:postgres_server) do
+      add_column :certificate_last_rotated_at, :timestamptz, null: false, default: Sequel.lit("now()")
```

Comment:
Yes this is logical server. Maybe I should pick another name for postgres_server indicating that it is logical while it is still early.

>  So, somewhere, whether it's two columns (like SSH) or a join, there needs to be slots for multiple roots of trust.
Makes sense. This only deals with the rotation of server certs because currently we create root certs for 10 years (btw do you have comment on the length? should it be shorter maybe?). But that time will come eventually so we should have a way to have multiple roots of trust.

Reply:
Yeah, maybe rename; it clashes too much with how we'd describe a physical server. I don't have a proposal. Even "postgres database" has some lexical clashing with the `create database` consruct, to say nothing of "postgres cluster.", which clashes with the `initdb` construct and...other things.

I seem to remember that industry practice is getting more cramped. Also, given our roots are not stored offline and all that other pentagony-stuff, we may want to choose a shorter interval and habituate customers to refreshing their trust root annually, at least in part because, in event of something horrible where they need to rotate it, we don't want everyone to be scramble with zero rehearsal that they've been putting off.

Reply:
After discussing this separately, we decided to start with 10 year validity period and revisit this decision in the future.

I'll do the renaming in a different PR (I don't have a good name ideas as well, I'll think about it).